src/proto/message_translation/tbots_protobuf.py

```diff
@@ -1,15 +1,15 @@
 from __future__ import annotations
 
 from proto.import_all_protos import *
-import software.python_bindings as tbots_cpp
+import software.python_bindings as cpp_bindings
```

Comment:
nit: I dont think these changes are all from ur PR, but it looks like you may have had some conflict with it so its worth noting.
We should stick with `tbots_cpp` for our python_bindings imports. It seems like both are used